### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by ezynda3**

### CODE REVIEW

File: docs/Deploy.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+- /scripts/deploy.sh (for deploying any of our contracts)
+  - execute with command `./scripts/deploy.sh` in your console
+- /scripts/diamond-update.sh (update / add to diamond)
+  - execute with command `./scripts/diamond-update.sh` in your console
+- /scripts/update-periphery.sh (add new contract to periphery registry)
+  - execute with command `./scripts/update-periphery.sh` in your console
+
+## Deploy facet and LiFiDiamond contracts
+
```
Comment (maxklenk) [inline docs/Deploy.md:45]: Perhaps add that the CREATE3_FACTORY has to be deployed to that chain first using the other repository.

**Review by maxklenk — COMMENTED** [review]
Nice, that helps. Just added two small comments.

File: docs/Deploy.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
+# Contract Deployment Checklist
+
+The following content will guide you through deploying and configuring all smart contracts to a new network (or to deploy a new staging environment).
+
+If you only want to deploy a new diamond contract and use existing deployed facets, please skip to [this section](#deploy-new)
+
+If you want to deploy an immutable diamond, please skip to this section
```
Comment (satoshi-lifi) [inline docs/Deploy.md:7]: maybe the link to the section is missed?

**Review by satoshi-lifi — APPROVED** [review]
LGTM

**Review by maxklenk — CHANGES_REQUESTED** [review]
When testing a deployment based on the configs I found some more points.

File: .env.example [hunk 3]
```diff
@@ -1,14 +1,50 @@
...
+ETH_NODE_URI_HARMONY=https://api.harmony.one
+ETH_NODE_URI_EVMOS=https://evmos-evm.publicnode.com
+ETH_NODE_URI_AURORA=https://1rpc.io/aurora
+ETH_NODE_URI_BOBA=https://mainnet.boba.network
+ETH_NODE_URI_GOERLI=https://rpc.ankr.com/eth_goerli
+ETH_NODE_URI_SEPOLIA=https://rpc.sepolia.org
+ETH_NODE_URI_MUMBAI=https://rpc.ankr.com/polygon_mumbai
+ETH_NODE_URI_NOVA=https://nova.arbitrum.io/rpc
+
+#ETHERSCAN
```
[the remark this answers is not in the document]
Comment (ezynda3) [reply .env.example]: Verification doesn't work there yet

**Review by maxklenk — APPROVED** [review]

**Pull request merged**